.travis.yml

```diff
@@ -0,0 +1,18 @@
+---
+language: go
+sudo: false
+go:
+  - 1.7.3
+  - 1.8.1
+  - tip
```

Comment:
I think that if you use `1.7` and `1.8` it'll already use the latest minor version.

Also, not sure if we need to test `tip` here.

Reply:
Ops, also Go latest version is `1.10` so I think:

```yml
go:
  - '1.8'
  - '1.9'
  - '1.10'
```